RELEASE_NOTES.rst

```diff
@@ -16,6 +16,7 @@ Migration notes
 All changes
 -----------
 
+- Correct functionality-syntax of :meth:`.vintage_and_active_years` (:pull:`623`).
 - Extend functionality of :meth:`.vintage_and_active_years`; add aliases
   :meth:`.yv_ya`, :meth:`.ya`, and :attr:`.y0` (:pull:`572`).
```

Comment:
This is fine.

Just FYI, another option, when we have 2+ PRs in the same release that combine to create/extend the same feature(s), is to mention them all after the same bullet e.g.:
```rest
- Extend functionality of … (:pull:`572`, :pull:`623`)
```